src/cli/crd/create.test.ts

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023-Present The Pepr Authors
+
+import { expect, describe, it, jest, beforeEach } from "@jest/globals";
+import { validateScope, generateCRDScaffold } from "./create";
+
+describe("generateCRDScaffold", () => {
+  it("should generate the correct CRD scaffold", () => {
+    const group = "example";
+    const version = "v1alpha1";
+    const kind = "MyCustomResource";
+    const data = {
+      domain: "example.com",
+      plural: "mycustomresources",
+      scope: "Namespaced",
+      shortName: "mcr",
+    };
```

Comment:
Thoughts on pre-emptively turning this into a generator function? Something along the lines of `createDefaultCRD()`? We've started to adopt that pattern elsewhere in our test suite.

Reply:
Yes, we could do that. Here is an issue for it. Due to resource constraints this week. 

https://github.com/defenseunicorns/pepr/issues/2053

Reply:
Sounds good, happy to take that on as follow-on work